modules/vba_documents/spec/request/v1/uploads_request_spec.rb

```diff
@@ -126,6 +126,7 @@
         expect(response.status).to eq(404)
       end
     end
+
     it 'returns a 200 with content-type of zip' do
```

Comment:
@f1337 i _think_ its always this spec that fails, at least that's the one I've seen a lot. Don't know if it matters or not but seems like the retry should be as close to the source as possible.